src/Wsdl/Decorator/Cache/CacheDecorator.php

```diff
@@ -67,6 +67,12 @@ public function call(string $functionName, array $arguments = [])
         return $response;
     }
 
+	/**
+	 * @param string $functionName
```

Comment:
Neni ted ``@param`` pro  ``$functionName`` a ``return`` zbytecny?  Ve SkautisQuery jsi je odebiral kdyz u nich nebyl zadny komentar.

Reply:
ok, odeberu. On má každej jinej coding style, nebyl jsem si jistej jak je to tady.